### DISCUSSION TITLE
OTG Plug and Play Webcam compatibility

### DISCUSSION THREAD

**Discussion by Burgert3067** [feelfreelinux/octo4a#75] [Ideas]
Firstly, I want to say thank you for creating this work of art! Being on a somewhat tight budget, this has saved me the hassle of trying to scrounge up enough money for a pi and related hardware so I could have some peace of mind while my printer is running!

More to the point: All of the phones I currently have, including the one being used with octo4a, have broken cameras and are pretty temperamental being inside the hot enclosure, which often means that I can't see a thing 90% of the time, and the other 10% of the time my phone turns itself off to avoid exploding. I have a plug and play webcam that's fairly resistant to high temperature environments, and I would love if USB camera integration was added in addition to being able to use the native cameras as it would make it easier to tailor a setup to an individual's needs!

I've been fiddling around with different USB camera IP streaming apps, but to be honest most of them are quite unreliable and I don't entirely trust them!

Thanks for your time!

**Comment by Shinare73** [comment]
Hi I created github account so that I too could request webcam support.  This is running great on my LG G8 but when I plug in my logitech c270 it does not recognize.  Thank you so much for your greatness!

**Comment by imthekeepe** [comment]
Thank you so much for all your work on this amazing program! I also would like to add to this discussion.

So I have a Pixel 4 XL and I got a Y splitter and plugged in a usb Hub and then attached a Vitade 960A Webcam. Octoprint 4a sees the device as USB Live cam under printers and lets me select different serial drivers but I want to know if you can make it so that I can select this device under cameras so that I can have the phone outside of my hot enclosure?

I am happy to send you any information you need in order to make this addition easier.
